apps/src/aichat/views/modelCustomization/compare-models-dialog.module.scss

```diff
@@ -0,0 +1,100 @@
+@import 'color';
+
+.textContainer {
+  background-color: $light_gray_50;
+  border-radius: 4px;
+  padding: 2px 0;
+}
+
+.modelDescriptionContainer {
+  flex: 1;
+  padding: 5px 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+#modelComparisonDialog {
+  height: 600px;
```

Comment:
The 'Finish' button is getting cut off one of my screens. 

![Screenshot 2024-04-04 at 10 45 58 AM](https://github.com/code-dot-org/code-dot-org/assets/107423305/3550571f-5099-49f9-8fb1-cdc28bfb9645)

What do you think about using something like `calc(vh - 100px)`?

Reply:
TIL, thanks! Looks great, [updated here](https://github.com/code-dot-org/code-dot-org/pull/57775/commits/c1e2c52c293acaf04c459db359461b815baec4e8)